maintainers/maintainer-list.nix

```diff
@@ -2339,6 +2339,12 @@
     githubId = 15774340;
     name = "Thomas Depierre";
   };
+  diegolelis = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add diegolelis``.

Reply:
Done